stdlib/scripts/build-stdlib.sh

```diff
@@ -39,4 +39,11 @@ STDLIB_PACKAGE_NAME="stdlib.mojopkg"
 FULL_STDLIB_PACKAGE_PATH="${BUILD_DIR}"/"${STDLIB_PACKAGE_NAME}"
 mojo package "${STDLIB_PATH}" -o "${FULL_STDLIB_PACKAGE_PATH}"
 
+echo "Validating API documentation strings in the Standard Library."
+
+mojo doc validate "${STDLIB_PATH}" || {
```

Comment:
`mojo doc validate` doesn't exist, it's `mojo doc`. This is what causes the CI to fail.